Sources/ArcGISToolkit/Components/Compass/CompassBody.swift

```diff
@@ -22,8 +22,11 @@ struct CompassBody: View {
             Circle()
                 .inset(by: borderWidth / 2.0)
                 .stroke(lineWidth: borderWidth)
-                .foregroundColor(.outline)
-                .background(Circle().foregroundColor(.fill))
```

Comment:
This modifier was also deprecated, which is why the code was changed to use the non-deprecated closure-based replacement.